.github/workflows/template_gitops.yml

```diff
@@ -61,18 +56,13 @@ jobs:
         with:
           docker-username: ${{ secrets.docker-username }}
           docker-password: ${{ secrets.docker-password }}
-          # remove npm token, goproxy, gonosumdb if feature is available: https://github.com/github-community/community/discussions/17554
-          docker-build-args: |
-            ${{ inputs.docker-build-args }}
-            NPM_TOKEN=${{ secrets.npm-token }}
-            GOPROXY=${{ secrets.goproxy }}
-            GONOSUMDB=${{ secrets.gonosumdb }}
```

Comment:
I'm not sure about the gonosumdb ... basically this is no secret and for me it is still okay to use this as build argument. I guess we should still allow that

Reply:
I would maybe leave it as we defined it as company secret and then maybe that's a little more consistent? 🤔